### ISSUE TITLE
[BUG] 1.20 has broken our tests in WebKit (page is just blank)

### ISSUE THREAD

**Issue by jdwholmes** [microsoft/playwright#12823]
- Playwright Version: 1.20
- Operating System: Windows
- Node.js version: 16.3
- Browser: WebKit
- Extra: Since upgrading to 1.20 none of our tests work on WebKit anymore, downgrading the package works.

I have an example of the test here https://try.playwright.tech/?l=javascript&s=k7pbmkr

```
const playwright = require('playwright');

(async () => {
  // Try to add 'firefox' to the list ↓
  for (const browserType of ['chromium', 'webkit']) {
    /** @type {import('playwright').Browser} */
    const browser = await playwright[browserType].launch();
    const context = await browser.newContext();
    const page = await context.newPage();
    await page.goto('https://console.cloudmanagementtest.apteco.com/authentication/login', {
      waitUntil: "networkidle"
    });
    await page.screenshot({ path: `example-${browserType}.png` });
    await browser.close();
  }
})();

```

try.playwright.tech is currently on 1.19 and it works correctly, I can also confirm that it works correctly when I downgrade my package back to 1.19. However 1.20 doesn't load any content on the page.

**Comment by JuanCruzTrinidad** [comment]
Playwright Version: 1.20
Operating System: Linux
Node.js version: 16.3
Browser: Chromium

I have the same problem.
After doing the before all suite, it does not continue with the tests. As if it did not receive the load event

**Comment by pavelfeldman** [comment]
I can reproduce the Windows issue. Investigation notes:

Last good WK: 1609
First bad WK: 1610

Roll: browser(webkit): roll to 02/03/22 (#11852)
Good base: [dd0bc728b4618f059dafab7e742f261a174c3a70](https://github.com/WebKit/WebKit/commit/dd0bc728b4618f059dafab7e742f261a174c3a70)
Bad base: [f522f6f875f2443ba32dc2cce3d8b0cfd97ffc77](https://github.com/WebKit/WebKit/commit/f522f6f875f2443ba32dc2cce3d8b0cfd97ffc77)

**Comment by pavelfeldman** [comment]
> @JuanCruzTrinidad Please file a separate issue with the repro details!

**Comment by jnizet** [comment]
I have the same problem. A tests that passed with 1.19 now fails with 1.20. Here are some additional details:

I have a link looking like this:

```html
<a href="/activities/42"><h3>Foo</h3></a>
```

Using `await page.click('text=Foo');` fails on chromium, but passes on the other browsers.

Changing it to `await page.click('text=Foo', { force: true });` makes it pass on chromium, but then fails on webkit.

Changing it to `await page.click('a:has-text("Foo")');` makes it pass everywhere.

To make things even more confusing, the failure on webkit happens on Github actions (Ubuntu VM), but doesn't happen locally (macOS).

**Comment by pavelfeldman** [comment]
@jnizet also looks like a separate issue, could you file a separate bug?

**Comment by jnizet** [comment]
@pavelfeldman sorry, I meant to comment on  #12821, which is the issue I'm facing. I deleted my comment here and added it on #12821.

**Comment by pavelfeldman** [comment]
```
f522f6f875f2 REGRESSION(r288878) webkitpy.layout_tests.controllers.layout_test_finder_legacy_unittest.LayoutTestFinderTests tests are failing with Windows Python https://bugs.webkit.org/show_bug.cgi?id=236043
50649f30f3d9 Fix BSD compilation after r289056 https://bugs.webkit.org/show_bug.cgi?id=236086
15027a13f3d6 Rename NetworkConnectionToWebProcess::serverToContextConnectionNoLongerNeeded() https://bugs.webkit.org/show_bug.cgi?id=236088
4f6df5b94626 Web Inspector: [Flexbox] Add setting to guard Flexbox Inspector feature https://bugs.webkit.org/show_bug.cgi?id=236091 <rdar://problem/88443529>
fc2ca414866e Rename ServiceWorkerInitializationData / ServiceWorkerInformation so they can be reused by shared workers https://bugs.webkit.org/show_bug.cgi?id=236089
65b4a3eadfd6 Crash at com.apple.WebKit: WebKit::RemoteLayerBackingStore::display https://bugs.webkit.org/show_bug.cgi?id=236003 Source/WebCore:
691f589b94c1 Move and rename ServiceWorkerThreadProxy::setupPageForServiceWorker() https://bugs.webkit.org/show_bug.cgi?id=236085
603f9bc649d7 null ptr deref in RenderTreeBuilder::Block::attachIgnoringContinuation https://bugs.webkit.org/show_bug.cgi?id=234170
1b1ef468f38b [WebAuthn] Allow use of hardware-fixed credentials while using alternate store https://bugs.webkit.org/show_bug.cgi?id=235923 rdar://88102108
8ab2d283ed84 Implement AbortSignal.timeout() https://bugs.webkit.org/show_bug.cgi?id=236039
3ed8019c7dad Flaky scope.assertNoException() assertion in ~JSExecState when running workers WPT tests https://bugs.webkit.org/show_bug.cgi?id=235978
30d34ccaabf7 Fix undefined behavior when querying linux memory size https://bugs.webkit.org/show_bug.cgi?id=228280
a0d1f22f518d [iOS][WP] Add file-ioctl telemetry https://bugs.webkit.org/show_bug.cgi?id=236031 <rdar://88334007>
13da0e979239 Start connecting SharedWorker to the WebKit2 layer https://bugs.webkit.org/show_bug.cgi?id=236052
08fcc92a850f [iOS][GPUP] Add access to sysctl property https://bugs.webkit.org/show_bug.cgi?id=235973 <rdar://problem/88345212>
937bacf4c6d5 [GPUP] Add read access to sysctl properties https://bugs.webkit.org/show_bug.cgi?id=236018 <rdar://problem/88382949>
71b000c52853 [LFC][IFC] imported/w3c/web-platform-tests/css/css-writing-modes/text-align-vlr* tests are failing https://bugs.webkit.org/show_bug.cgi?id=236059
dd9e7911cf9e MediaRecorderPrivate should not need to create IOSurfaces https://bugs.webkit.org/show_bug.cgi?id=235953
529237a6d794 RealtimeIncomingVideoSourceCocoa should not need to create IOSurfaces https://bugs.webkit.org/show_bug.cgi?id=235952
c0724ab78ee6 Incorrect KeyframesEffect generated for background https://bugs.webkit.org/show_bug.cgi?id=229398 <rdar://problem/82516118>
79a7de25ae34 Unreviewed. Add another missing file after r288871.
a6f290cc6a13 [GTK][a11y] Test /webkit/WebKitAccessibility/accessible/children-changed times out https://bugs.webkit.org/show_bug.cgi?id=236070
29739b4f6b95 [GTK][WPE][a11y] Handle parameter of org.a11y.atspi.Text.RemoveSelection https://bugs.webkit.org/show_bug.cgi?id=236079
2e8f2e564514 Crash in AXIsolatedObject::associatedAXObject. https://bugs.webkit.org/show_bug.cgi?id=236053 <rdar://problem/88413232>
a66b7567fcde Add specialize type traits to GLContext https://bugs.webkit.org/show_bug.cgi?id=236076
7eb496fa7749 WebDriver: selenium tests are executed more than once https://bugs.webkit.org/show_bug.cgi?id=236075
e91fc1aa7937 [atspi] missing-braces clang warnings https://bugs.webkit.org/show_bug.cgi?id=236074
ee5f8d5befda Unreviewed. Add missing files after r283798.
3cf3cd2e5992 Unreviewed. Add missing file after r288871.
77909c751bc1 SampleBufferDiplayLayer should not need to create IOSurfaces https://bugs.webkit.org/show_bug.cgi?id=235954
f5f497d7b7f4 [JSC] Use MetadataTable instead of UnlinkedMetadataTable https://bugs.webkit.org/show_bug.cgi?id=236063 rdar://88269480
8b5b81829963 Unreviewed. Update OptionsGTK.cmake and NEWS for 2.35.2 release
342b9f084275 Allow using STIX Two in layout tests https://bugs.webkit.org/show_bug.cgi?id=236048
658e3ab115fa Unreviewed. Fix GTK make distcheck
7f1c2ffe6a36 Transform interpolation should blend between shared transform function primitives https://bugs.webkit.org/show_bug.cgi?id=235311 <rdar://problem/88012700>
b6969911154d Create PDFDocument stub https://bugs.webkit.org/show_bug.cgi?id=235970
bf74097a3b15 Unreviewed. [GTK] Apply the same cmake changes of r289029 to GTK port
3a1756443c95 [WPE] Fix for non-unified builds with ACCESSIBILITY=OFF https://bugs.webkit.org/show_bug.cgi?id=236024
ef2bc68007b9 [GTK][a11y] Set platform name to atspi in WTR https://bugs.webkit.org/show_bug.cgi?id=232227
6f88cfb09896 [XCBuild] Users of libgtest.a depend on headers from gtest.framework https://bugs.webkit.org/show_bug.cgi?id=236045
6173bb6fcabc Remove debug log accidentally added with r289018 https://bugs.webkit.org/show_bug.cgi?id=236000
192fa243acfe [MiniBrowser][Win][WK2] Show MessageBox for alert, confirm and prompt https://bugs.webkit.org/show_bug.cgi?id=236047
2122e09199c1 SecureARM64EHashPins should check g_jscConfig.useFastJITPermissions https://bugs.webkit.org/show_bug.cgi?id=236055 <rdar://88222677>
05a4b3682427 Move connect event dispatch logic to SharedWorkerGlobalScope https://bugs.webkit.org/show_bug.cgi?id=236038
175276a2943f CSP: Implement wasm-unsafe-eval https://bugs.webkit.org/show_bug.cgi?id=235408
867e8efe8c9f CSP: Fix matching wildcard hosts https://bugs.webkit.org/show_bug.cgi?id=235992
f5ef87d86ce6 Speed-up JSON.stringify() by avoiding "toJSON" property lookups https://bugs.webkit.org/show_bug.cgi?id=235996
c33b8ca8ee9a Use private relay for all subresources if the main document main resource was loaded using private relay https://bugs.webkit.org/show_bug.cgi?id=236000 Source/WebCore:
2841b97d0d89 [Xcode] Fix redundant execution of "Check VTables..." script https://bugs.webkit.org/show_bug.cgi?id=236032
6ad58508c947 [GLIB] Unreviewed debug buildfix after r288872 https://bugs.webkit.org/show_bug.cgi?id=236036
dc851b79b400 [EWS] Rename CompileWebKitWithoutPatch to CompileWebKitWithoutChange https://bugs.webkit.org/show_bug.cgi?id=235949 <rdar://problem/88325688>
aa8a88ad9b1e Keyframe resolution methods should use reference instead of pointer parameters https://bugs.webkit.org/show_bug.cgi?id=236020
cdf8c64884ff Improve color contrast of dark mode syntax highlighting https://bugs.webkit.org/show_bug.cgi?id=236041
9888b1ab029c Remove OriginStorageManager if it's not in use https://bugs.webkit.org/show_bug.cgi?id=236029
37229c03326d Percentage word spacing doesn't incorporate synthetic bold expansion https://bugs.webkit.org/show_bug.cgi?id=236007
8626428282df [JSC] Crash on several pages after r287986 https://bugs.webkit.org/show_bug.cgi?id=236033
00a99214e09a Move TransferredMessagePort to its own header https://bugs.webkit.org/show_bug.cgi?id=236028
0eeac5ccba5d Web Inspector: didShowExtensionTab needs to pass the frameID of the tab https://bugs.webkit.org/show_bug.cgi?id=236023 rdar://problem/88383984
4a5e6b91d387 WebInspector: Add additional image MIME types to frontend https://bugs.webkit.org/show_bug.cgi?id=235977
5b5021dc115e No breakpoints hit on github.com, and some are invalid https://bugs.webkit.org/show_bug.cgi?id=235607
237db45bb237 [JSC] YarrJIT m_checkedOffset should be pre-computed and stored to Yarr op https://bugs.webkit.org/show_bug.cgi?id=235932
aad69b646b64 [CMake] Stop defaulting to ld.gold https://bugs.webkit.org/show_bug.cgi?id=235476
992d2e5722ef [JSC] Remove compiler warning in TrailingArray.h https://bugs.webkit.org/show_bug.cgi?id=236014
a70089b9ddf2 CSP: Fix returned WebAssembly error type when blocked https://bugs.webkit.org/show_bug.cgi?id=235319
e9dabfdfb993 [GPU Process] Remove DisplayList::Replayer::Delegate and DisplayList::RecorderImpl::Delegate https://bugs.webkit.org/show_bug.cgi?id=235939
f529867994c7 [GLIB] Unreviewed debug buildfix after r288872 https://bugs.webkit.org/show_bug.cgi?id=236036
a2d149f2c8ec Unmute link errors for ES module scripts https://bugs.webkit.org/show_bug.cgi?id=235925
5dc8c5604d12 Bring up additional test queues on Big Sur https://bugs.webkit.org/show_bug.cgi?id=235446
f4ebef5cf87e Fixes for non-unified builds after r288868 https://bugs.webkit.org/show_bug.cgi?id=235984
f085b2650f61 REGRESSION(r288865): SourceImage should never sink its ImageBuffer to a NativeImage https://bugs.webkit.org/show_bug.cgi?id=236005
f25e0097ebb9 Move SharedWorkerThread & SharedWorkerProxy to workers/shared/context/ https://bugs.webkit.org/show_bug.cgi?id=236017
99275d5ddcce Support sending WorkerOptions over IPC https://bugs.webkit.org/show_bug.cgi?id=236026
12df6237b845 Enable overflow:clip feature https://bugs.webkit.org/show_bug.cgi?id=236008
c4e9bdf488e4 Move and rename ServiceWorkerFrameLoaderClient & ServiceWorkerLibWebRTCProvider https://bugs.webkit.org/show_bug.cgi?id=236016
5d9e614c582f [GStreamer] Generate webrtc encoder src pad template based on supported platform encoders https://bugs.webkit.org/show_bug.cgi?id=235964
816d8534ba59 Update computation of FAST_TLS base. https://bugs.webkit.org/show_bug.cgi?id=235934
3d81614767e4 [git-webkit] Improve user prompts (Part 5) https://bugs.webkit.org/show_bug.cgi?id=235655 <rdar://problem/88082697>
9de7e6607557 [git-webkit] Improve user prompts (Part 4) https://bugs.webkit.org/show_bug.cgi?id=235655 <rdar://problem/88082697>
6484f5aeb96f [git-webkit] Improve user prompts (Part 3) https://bugs.webkit.org/show_bug.cgi?id=235655 <rdar://problem/88082697>
22e86aa493d2 [git-webkit] Improve user prompts (Part 2) https://bugs.webkit.org/show_bug.cgi?id=235655 <rdar://problem/88082697>
1632b86d3e22 [git-webkit] Improve user prompts (Part 1) https://bugs.webkit.org/show_bug.cgi?id=235655 <rdar://problem/88082697>
788d17cba5a3 CSSPropertyAnimation::animatableShorthandsAffectingProperty() is unused https://bugs.webkit.org/show_bug.cgi?id=236022
e7b58fbc79e9 Make modifications to the isolated Tree atomic. https://bugs.webkit.org/show_bug.cgi?id=235683 <rdar://problem/88108412>
f1a3f1e8534b REGRESSION (r288869): webkitpy.w3c.test_converter_unittest.W3CTestConverterTest.test_convert_prefixed_properties is failing https://bugs.webkit.org/show_bug.cgi?id=235961 <rdar://problem/88332029>
9788e4d0118d Unreviewed, fix the macOS build after r288956 https://bugs.webkit.org/show_bug.cgi?id=235991
9914d54e3934 null ptr deref in Editor::pasteWithPasteboard https://bugs.webkit.org/show_bug.cgi?id=235966
b5f95d22b563 WPT: Import WebAssembly CSP tests https://bugs.webkit.org/show_bug.cgi?id=235319
383c81d663d9 [EWS] unit tests have become slower https://bugs.webkit.org/show_bug.cgi?id=235364 <rdar://problem/88073408>
6b0c73fbeb5f [iOS][WP] Add access in sandbox to Mach message https://bugs.webkit.org/show_bug.cgi?id=236021 <rdar://88300200>
4acc4dec4669 Adjust some Live Text code to only use VKC-prefixed VisionKit classes if ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS) https://bugs.webkit.org/show_bug.cgi?id=235991
e6b0c475c269 Unreviewed, reverting r286988. https://bugs.webkit.org/show_bug.cgi?id=236006
b23b16a09fc6 RealtimeVideoSource::adaptVideoSample should not create IOSurfaces when running in WebProcess https://bugs.webkit.org/show_bug.cgi?id=236011 <rdar://88367968>
c33c1d78649c AX: Display startObject and anchorObject objectID when logging AccessibilitySearchCriteria from AXLogger https://bugs.webkit.org/show_bug.cgi?id=235997
36878c3756ef [ews] Read buildbot try credentials from passwords.json instead of env variables https://bugs.webkit.org/show_bug.cgi?id=236015
42e6d5f2a04e LibWebRTCCodecs should not need to create IOSurfaces https://bugs.webkit.org/show_bug.cgi?id=235951 <rdar://problem/88326654>
f35344716440 Clarify that some UUID routines are dedicated to UUID v4 https://bugs.webkit.org/show_bug.cgi?id=235430
0a8589b383e4 ServiceWorkerNavigationPreloader should only be used once https://bugs.webkit.org/show_bug.cgi?id=235882 <rdar://88226432>
150ec89edc77 [GStreamer] Add support for custom message handling in simpleBusMessageCallback https://bugs.webkit.org/show_bug.cgi?id=235822
6c356649b320 scroll-margin-top doesn't work on inline elements https://bugs.webkit.org/show_bug.cgi?id=235933 <rdar://87983307>
dd2b33e58ded [JSC] wasm atomic opcodes should be rejected if alignment is not equal to natural width https://bugs.webkit.org/show_bug.cgi?id=235990
53d02bc8408a [LFC][IFC] Using Fontcascade::spaceWidth to subtract the trailing space width may result in incorrect layout https://bugs.webkit.org/show_bug.cgi?id=235937 <rdar://88059633>
2e974bceace0 Update watchOS simulators to run on the newer 44mm display https://bugs.webkit.org/show_bug.cgi?id=235986
f168beae6dfa Rename FontCascade::fontMetrics() and RenderStyle::fontMetrics() to fontMetricsOfPrimaryFont() https://bugs.webkit.org/show_bug.cgi?id=235938
505f6ed4699d Unreviewed, non-unified buildfix after latest accessibility commits https://bugs.webkit.org/show_bug.cgi?id=235999
8a961b890dc8 Share more WebPage code related to marking layers volatile https://bugs.webkit.org/show_bug.cgi?id=235985
41504bafc77f Unreviewed, reverting r288914. https://bugs.webkit.org/show_bug.cgi?id=235998
317f699fcf1b Rename ServiceWorkerFetchResult to WorkerFetchResult and make it more generic https://bugs.webkit.org/show_bug.cgi?id=235983
c39ea7f39844 Check AccessibilityRenderObject::m_renderer for null before using it. https://bugs.webkit.org/show_bug.cgi?id=235950 <rdar://problem/88326438>
99bd64fbb011 Replace WebCore::topOriginURL() to SecurityOrigin::toURL() https://bugs.webkit.org/show_bug.cgi?id=235987
da238dec54eb Rename some service worker functions that have names that are too generic https://bugs.webkit.org/show_bug.cgi?id=235988
3b925674c769 The default values for a couple of Live Text-related features should respect system feature flags https://bugs.webkit.org/show_bug.cgi?id=235980 rdar://88349726
d1b703ceb781 Crash under ScrollingCoordinatorMac::hasNodeWithAnimatedScrollChanged() https://bugs.webkit.org/show_bug.cgi?id=235989 <rdar://87402941>
3d85882f215a Fix build error with VIDEO=ON https://bugs.webkit.org/show_bug.cgi?id=235982
fde694156283 Add setting to toggle PDF.js viewer https://bugs.webkit.org/show_bug.cgi?id=235972
1ab7da4d4ae0 Forward messages from NetworkProcess to webpushd https://bugs.webkit.org/show_bug.cgi?id=235855
a0c1449fc8b8 Mitigate hangs underneath -requestAutocorrectionContextWithCompletionHandler: when focusing text fields https://bugs.webkit.org/show_bug.cgi?id=235963 rdar://70152487
69ae9efe43bc Write origin file when OriginStorageManager is destroyed https://bugs.webkit.org/show_bug.cgi?id=235897 rdar://87163253
c9329d7ac444 [WP] Remove PostScript conversion code https://bugs.webkit.org/show_bug.cgi?id=235892 <rdar://88172449>
e991f73f586a Catalyst builds of WebKit with system content path put XPCServices and Daemons in the normal location https://bugs.webkit.org/show_bug.cgi?id=235976
1e32607bb48e [Flatpak SDK] Using thin archives with ld.bfd results in linker errors https://bugs.webkit.org/show_bug.cgi?id=235975
9be116bad1f1 [LFC][IFC] Using Fontcascade::spaceWidth to subtract the trailing space width may result in incorrect layout https://bugs.webkit.org/show_bug.cgi?id=235937 <rdar://88059633>
10f9230ccffd MSVC reports "SVGPropertyAnimator.h(94): error C2839: invalid return type 'T *' for overloaded 'operator ->'" with /std:c++20 https://bugs.webkit.org/show_bug.cgi?id=234546 <rdar://problem/86757805>
6cffb625a25a Use more AtomString and un-inline code for ScriptElementCachedScriptFetcher and its subclasses https://bugs.webkit.org/show_bug.cgi?id=235919 <rdar://problem/88338714>
6a28abd1d93f [EWS] Fix infinate retry on PR with red ToT https://bugs.webkit.org/show_bug.cgi?id=235971 <rdar://problem/88344438>
e49dcd98e4c5 [macOS][WP] Add required syscall https://bugs.webkit.org/show_bug.cgi?id=235828 <rdar://88177392>
49b25ac5479e Refactor KeyframesRuleMap to use AtomString for the key https://bugs.webkit.org/show_bug.cgi?id=235962
c27ac2ca4794 [CSS Container Queries] Ensure query containers have valid layout before resolving the subtree https://bugs.webkit.org/show_bug.cgi?id=235896
15b8dbda0d5d [GTK][WPE] Fixes for non-unified builds after r288807 and r288820 https://bugs.webkit.org/show_bug.cgi?id=235956
22a699e0890b Import APSConnection-related SPI https://bugs.webkit.org/show_bug.cgi?id=235856
a1ac289ee114 Unreviewed, reverting r288538. https://bugs.webkit.org/show_bug.cgi?id=235968
d8e8c39ff82b Use more AtomString and un-inline code for ScriptElementCachedScriptFetcher and its subclasses https://bugs.webkit.org/show_bug.cgi?id=235919
ef6725945a09 Unreviewed, reverting r288746. https://bugs.webkit.org/show_bug.cgi?id=235967
1559425ade7d Protect NetworkConnectionToWebProcess::createFetchTask from a null service worker connection https://bugs.webkit.org/show_bug.cgi?id=235878 <rdar://87785042>
5da8d58eb9a7 [Cocoa] "index 0 out of bounds" exception in AVTrackPrivateAVFObjCImpl::videoTrackConfiguration() https://bugs.webkit.org/show_bug.cgi?id=235918 <rdar://87651247>
5c80a5ce0703 [WebXR] Handle changes in XRSession's visibility state https://bugs.webkit.org/show_bug.cgi?id=235898
e83f97e11aa5 [macOS] Expose screen and window capture state to WKWebView https://bugs.webkit.org/show_bug.cgi?id=235838 <rdar://problem/88207924>
bbd1c3f1f50c null ptr deref in LayoutIntegrationLineLayout LineLayout::adjustForPagination and LineLayout::visualOverflowBoundingBoxRectFor https://bugs.webkit.org/show_bug.cgi?id=235907
91f70aad6e7c Enhance sanitizeStackForVM() to assist with crash analysis. https://bugs.webkit.org/show_bug.cgi?id=235752 rdar://81014601
b67ad67ce867 [GStreamer] Update flags in internal GStreamer source and sink elements https://bugs.webkit.org/show_bug.cgi?id=235871
53df3a5c83f7 [GStreamer] Support for vpx in webrtc encoder https://bugs.webkit.org/show_bug.cgi?id=235870
bd0b8f769cce Redefining @keyframes does not work https://bugs.webkit.org/show_bug.cgi?id=229437 <rdar://problem/82563372>
977be0190efb Animation from scale(0) has missing backing store https://bugs.webkit.org/show_bug.cgi?id=227733 <rdar://problem/80246192>
1a8ec6d663e0 Minor cleanup in RemoteLayerBackingStoreCollection https://bugs.webkit.org/show_bug.cgi?id=235936
a531ebe38589 [css-cascade] Fix removal of not yet loaded CSS @import https://bugs.webkit.org/show_bug.cgi?id=235930
0da918f71ea3 Migrate LayoutTestFinder tests from FileSystemMock to pyfakefs https://bugs.webkit.org/show_bug.cgi?id=235370 <rdar://problem/87785236>
7bf7d10638b9 RemoteRenderingBackendProxy should not be a IPC::MessageSender https://bugs.webkit.org/show_bug.cgi?id=235935
5a8cf276d8fb [BigSur+ wk2 Debug] webrtc/video-mute-vp8.html is a flaky failure https://bugs.webkit.org/show_bug.cgi?id=230359 <rdar://problem/83203317>
d215bd0bbdd7 [EWS] Trigger embedded builders on pull-requests https://bugs.webkit.org/show_bug.cgi?id=235913 <rdar://problem/88290964>
c51d4f7935b1 AX: nullptr crash under AccessibilityRenderObject::computeAccessibilityIsIgnored https://bugs.webkit.org/show_bug.cgi?id=235945 rdar://88217815
7fcd24164350 Unreviewed. [GTK][WPE] Fix compile warning
e52f160f81b8 [GTK][WPE] Switch to use ATSPI instead of ATK https://bugs.webkit.org/show_bug.cgi?id=235942
ae8ab5c6ae19 Unreviewed. Update W3C WebDriver imported tests.
0a8e2e34acd9 Revert OSAllocator behavior to pre-Structure-Allocator change one https://bugs.webkit.org/show_bug.cgi?id=235940
0329e84f139f Entirely remove support for -apple-trailing-word https://bugs.webkit.org/show_bug.cgi?id=234672
f55df6b6048f Rename FontCascade::fontMetrics() and RenderStyle::fontMetrics() to fontMetricsOfPrimaryFont() https://bugs.webkit.org/show_bug.cgi?id=235938
fb4da5fb18c6 Fix handling of access key events https://bugs.webkit.org/show_bug.cgi?id=234147 <rdar://problem/86320218>
f2204048d6bc [libpas] get_num_free_bytes_for_each_heap_callback() is called with `arg` pointing to uninitialized stack memory <https://webkit.org/b/235931> <rdar://problem/88303899>
04f246f532ec [GPU Process] Pattern should hold SourceImage which can be converted to a NativeImage only when needed https://bugs.webkit.org/show_bug.cgi?id=232411 <rdar://problem/84998738>
1f400b5e11e1 [iOS] Don't register undo actions in undoManager overridden by WKWebView client https://bugs.webkit.org/show_bug.cgi?id=235928 <rdar://problem/87192657>
0dd304ecd027 REGRESSION(288052?): editing/execCommand/paste-as-quotation-disconnected-paragraph-ancestor-crash.html makes subsequent test fail, as DumpRenderTree compares to wrong expected result https://bugs.webkit.org/show_bug.cgi?id=235733
f4921760fe6e [iOS] Clean up some issues in the 'com.apple.WebKit.adattributiond.sb' sandbox https://bugs.webkit.org/show_bug.cgi?id=235929 <rdar://88081007>
67b9bd5ca33b Add to AXLogger the ability to log only from the specified thread. https://bugs.webkit.org/show_bug.cgi?id=235843 <rdar://problem/88211205>
c88c969be3c7 [RISCV64] Add remaining MacroAssemblerRISCV64 branching operations https://bugs.webkit.org/show_bug.cgi?id=235886
8e3f1d095ebc Try-reserving Structure heap after reserving all the other heaps including JIT heap https://bugs.webkit.org/show_bug.cgi?id=235922
fa26860f3f98 Collapsed ranges return zero rects in DocumentContext https://bugs.webkit.org/show_bug.cgi?id=235844
c3ec1dd09953 Web Inspector: [Cocoa] Crash in WebKit::WebInspectorUIProxy::attachmentViewDidChange https://bugs.webkit.org/show_bug.cgi?id=235842
4b4206d9a657 [GPU Process] Remove the resource use counters from the remote resource cache https://bugs.webkit.org/show_bug.cgi?id=235636
f41088fcd852 WindowServer returned not alive with context:,unresponsive work processor(s) rdar://problem/86037417
f7498ae10d0b Revert "WindowServer returned not alive with context:,unresponsive work processor(s)"
873cae1c7519 [macOS][WP] Add required syscall https://bugs.webkit.org/show_bug.cgi?id=235901 <rdar://88241768>
590f5408cf0a Delete runtime flag for subresource integrity https://bugs.webkit.org/show_bug.cgi?id=235876
35467df0470c Versioning.
1c0ff93e7cec [RISCV64] Fix offlineasm lowering of arithmetic branch instructions with address destinations https://bugs.webkit.org/show_bug.cgi?id=235887
149d26dc5b36 [EWS] Revert pull request when retrying https://bugs.webkit.org/show_bug.cgi?id=235825 <rdar://problem/88187688>
b4a152ff76f8 [Mac] Fix TestWebKitAPI build https://bugs.webkit.org/show_bug.cgi?id=235906
e6be01c0712c [iOS] SharePlay is shown prominently for system-provided Share Sheets https://bugs.webkit.org/show_bug.cgi?id=235895 rdar://88268450
115486a3f853 LineLayout::collectOverflow() should null check m_inlineContent https://bugs.webkit.org/show_bug.cgi?id=235902 rdar://87402815
1f9b42f7b4bf Source/WebCore: Changed playbackRate and defaultPlaybackRate from unrestricted double to double. https://bugs.webkit.org/show_bug.cgi?id=235174 rdar://problem/87815745
c5f7467016b8 [iOS][WP] Restrict image decoders https://bugs.webkit.org/show_bug.cgi?id=234175 <rdar://problem/86650455>
e3f64e0c4b6c [WP] Remove PostScript conversion code https://bugs.webkit.org/show_bug.cgi?id=235892 <rdar://88172449>
92ba5246ef27 Fix handling of module script errors in WebCore https://bugs.webkit.org/show_bug.cgi?id=218284
5b0cb55ac33e [iOS][WP] Restrict image decoders https://bugs.webkit.org/show_bug.cgi?id=234175
db4bcc64162e [WebAuthn] Provide SPI to export/import local credentials https://bugs.webkit.org/show_bug.cgi?id=234112 rdar://84822000
c888e174bc83 Reland StructureID overhaul https://bugs.webkit.org/show_bug.cgi?id=235720
ef9e9e97b731 Support additional WPEToolingBackend types https://bugs.webkit.org/show_bug.cgi?id=235745
215039ef09d6 [CSS Container Queries] Check for query containers when matching rules https://bugs.webkit.org/show_bug.cgi?id=235881
23827533f9f7 [GTK][a11y] WTR: use AXMenuItemSelected for menu item selection change with ATSPI https://bugs.webkit.org/show_bug.cgi?id=235890
c24904716924 [ews] Cleanup code by removing patchFailedToBuild and patchFailedTests properties https://bugs.webkit.org/show_bug.cgi?id=235860
cdaedbaf2eda [Flatpak SDK] No longer updating due to expired GPG key https://bugs.webkit.org/show_bug.cgi?id=235869
5b5aac2e0b83 REGRESSION(r288644): [GTK4] Criticals when using pinch zoom https://bugs.webkit.org/show_bug.cgi?id=235883
4019bc2e2cb7 [GTK][a11y] Add support for password fields to ATSPI https://bugs.webkit.org/show_bug.cgi?id=235778
d2c8a8a4144f [WTF] Add GenericHashKey https://bugs.webkit.org/show_bug.cgi?id=235872
3e631739e295 Update LayoutTests/resources/testharness.js to 2021-6-24 version https://bugs.webkit.org/show_bug.cgi?id=235398 <rdar://problem/88126222>
```

**Comment by pavelfeldman** [comment]
Fix coming up in 1.21 shortly.
